comms/dht/src/proto/mod.rs

```diff
@@ -22,7 +22,7 @@
 
 use crate::proto::envelope::Network;
 use std::fmt;
-use tari_crypto::tari_utilities::hex::Hex;
```

Comment:
tari_utilities was re-exported from tari_crypto so that one didn't have to explicitly include it

Reply:
I'm not sure I agree with always having to use the re-export, I'm using features of tari_utilities independent of tari_crypto. Re-exports are IMO useful if you cannot use a crate (e.g. tari_crypto) without the dependent crate (e.g. tari_utilities), which is the reason this re-export was included in the first place. However, the DHT is using features of tari_utilities that have nothing to do with tari_crypto and so is dependent on it. It's also kind of a pain because intellij doesn't import it properly :P

Another example: I re-exported Multiaddr from comms. The purpose of this is to prevent the need for all crates that use comms to have multiaddr as a dependency just to use multiaddresses. However, if a crate needs mutiaddress independently of comms, they should consider adding that as a direct dependency ("if comms stops being a dependency, will I still need multiaddr?").